### PR TITLE
Aswin file v2

### PR DESCRIPTION
**Pull request by Aswin31671**
Format changed

### CODE REVIEW

File: filehandling.c [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	        printf("Name:");
+                scanf("%s", emp[index].employeeName);
+                printf("Id:");
+                scanf("%d", &emp[index].employeeId);
+        }
+        fp=fopen("file1.bin", "wb");
+        if(fp == NULL){
+		printf("error in open file in write mode");
+        }
+        else{
```
Comment (tklogesh) [inline filehandling.c:26]: align the else part

File: filehandling.c [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		for(index = 0; index < n; index ++){
+        	fwrite(&emp, sizeof(struct Employee), n, fp);
+		}
+	fclose(fp);	
+        }
+	fp = fopen("file1.bin", "rb");
+        if(fp == NULL){      
+		printf("error open filn in read mode");
+	}
+	else{
```
Comment (tklogesh) [inline filehandling.c:36]: align the else part

File: filehandling.c [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                printf("Id:");
+                scanf("%d", &emp[index].employeeId);
+        }
+        fp=fopen("file1.bin", "wb");
+        if(fp == NULL){
+		printf("error in open file in write mode");
+        }
+        else{
+		for(index = 0; index < n; index ++){
+        	fwrite(&emp, sizeof(struct Employee), n, fp);
```
Comment (P-Prajwal) [inline filehandling.c:28]: Shouldn't this index to each employee? What's the use of `for` loop here otherwise

File: filehandling.c (hunk 1 above)
Comment (Aswin31671) [reply filehandling.c:26]: changed .

File: filehandling.c (hunk 3 above)
Comment (Aswin31671) [reply filehandling.c:28]: What's the use of for loop here otherwise?
write the  each employee name and id into the file so I used  for loop here.

Comment (P-Prajwal) [reply filehandling.c:28]: For that shouldn't u use `&emp[index]` ?

Comment (preeja398) [reply filehandling.c:28]: Just wondering how it worked. @Aswin31671  did u able to see expected output with this?

Comment (Aswin31671) [reply filehandling.c:28]: now  I  changed in new pull request

**Pull request closed without merging**